ui/app/styles/components/popover-menu.scss

```diff
@@ -0,0 +1,50 @@
+.popover-content {
+  border: 1px solid $grey-blue;
+  box-shadow: 0 6px 8px -2px rgba($black, 0.05), 0 8px 4px -4px rgba($black, 0.1);
+  margin-right: -$radius;
+  margin-top: -1px;
+  border-radius: $radius;
+  padding: 0.5em 1em;
+  width: 200px;
+}
+
+.popover-actions {
+  margin: 1rem -1rem -0.5rem -1rem;
+  border-top: 1px solid $grey-lighter;
+  display: flex;
+
+  .popover-action {
+    border: none;
+    height: 2.75em;
+    width: 100%;
+    margin: 4px;
+    border-radius: $radius;
+    font-size: 1rem;
+    font-weight: $weight-bold;
+    align-items: center;
+    cursor: pointer;
+
+    &:hover {
+      background: $white-ter;
+    }
+
+    &:focus {
+      background: $white-ter;
+      box-shadow: inset 0 0 0 2px rgba($blue, 0.5);
+      outline: none;
+    }
+
+    @each $name, $pair in $colors {
+      $color: nth($pair, 1);
+
+      &.is-#{$name} {
+        color: $color;
+      }
+    }
+
+    // Override is-primary, which is normally green, to be blue
+    &.is-primary {
+      color: $blue;
+    }
+  }
+}
```

Comment:
Maybe some `z-index` action is needed here? This happened when I scrolled with the popover open:

<img width="370" alt="image" src="https://user-images.githubusercontent.com/43280/70473472-bf974580-1a96-11ea-9455-8dddf0361c05.png">

Reply:
Fixed!